Backend/src/middlewares/passportOauth.ts

```diff
@@ -1,37 +1,44 @@
-import passport from 'passport';
-const path = require('path')
-require('dotenv').config({ path: path.join(process.cwd(), '.env') })
-const GoogleStrategy = require('passport-google-oauth').OAuth2Strategy;
-
+import passport from "passport";
+const path = require("path");
+require("dotenv").config({ path: path.join(process.cwd(), ".env") });
+const GoogleStrategy = require("passport-google-oauth").OAuth2Strategy;
```

Comment:
any good sources to read up on this?

Reply:
I used this a lot [passportjs.org](http://www.passportjs.org/docs/google/) and the other links i shared on discord. It was mostly a "see-what-fits-together" rather than deep understanding :/